test/test_module_tracker.py

```diff
@@ -76,6 +76,64 @@ def test_bw_detection(self):
             self.assertFalse(tracker.is_bw)
             self.assertEqual(tracker.parents, {"Global"})
 
+    def test_user_hooks(self):
+        class Bar(torch.nn.Module):
+            def __init__(self):
+                super(__class__, self).__init__()
```

Comment:
```suggestion
                super().__init__()
```
Not needed anymore in 3+